setup.py

```diff
@@ -7,22 +7,33 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
-with open('requirements.txt') as f:
-    required = f.read().splitlines()
-
 setup(
     name = "mcfly",
     version = __version__,
     description = ("Deep learning for time series data"),
-    license = "Apache 2.0",
+    license = "Apache Software License 2.0",
     keywords = "Python",
     url = "https://github.com/NLeSC/mcfly",
     packages = find_packages(),
-    install_requires=required,
     long_description=read('README.md'),
     long_description_content_type='text/markdown',
     classifiers=[
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
     ],
+    test_suite="tests",
+    python_requires='>=3.6',
+    install_requires=[
```

Comment:
What is the reason for including the requirements in the setup.py file, instead of reading the requirements.txt file? The consequence is that the requirements are now duplicated (unless we remove the requirements.txt file)

Reply:
There are some ideas on how to use requirements and setup.py. Some say that requirements.txt should be very specific, as in, the packages used for development of the tool, and setup stating the dependencies in a way that is as lenient as possible in order not to force specific versions. We chose a different approach at the time. We just wanted to get rid of the duplication. Changing to the other approach should only be done if we (or at least some of us) would agree on doing so I think. I still like our approach of not duplicating the dependency list.

Reply:
The idea was to have the necessary requirements as well as the additional requirements for testing (pytest, coverage...) defined at only one central location: `setup.py`. I now hence removed the `requirements.txt` and made adapted the `environment.yml` to take what is specified in `setup.py`. I think that reduce redundancies as far as possible. 
Let me know if you have a strong preference for having a separate requirement.txt file (than I can try making that work instead).

Reply:
I think this approach is fine. if we ever see the need to also have a `requirements.txt` file, we can simply add it with `.` as content, but I don't see the added value of that right now.